tests/e2e/test_as_tenant.yml

```diff
@@ -143,6 +196,7 @@
       tags: always
       vars:
         set_boot_disk_copy: true
+        set_no_data_copy: false
```

Comment:
And here it should be `set_no_data_copy: true`. Then i think the tests will behave as the comments here say.